### PR TITLE
VZ- 8353 LRE pipeline using Rancher Proxy

### PR DESCRIPTION
**Pull request by mbenmclean**
This change updates the LRE pipeline to access the cluster through the Rancher Proxy rather than through the bastion

### CODE REVIEW

File: ci/lre/Jenkinsfile [hunk 1]
```diff
@@ -190,12 +130,28 @@ pipeline {
             }
         }
 
-        stage('Upgrade to stable commit') {
-            when {
```
Comment (bobdonat) [inline ci/lre/Jenkinsfile:194]: Why was this when section removed?

File: ci/lre/Jenkinsfile [hunk 2]
```diff
@@ -190,12 +130,28 @@ pipeline {
...
-                expression {
-                    return params.UPGRADE_VZ
+        stage("Create OCI config file") {
```
Comment (davisgomes) [inline ci/lre/Jenkinsfile:133]: Why do we still need to set up the OCI config file?

Comment (mbenmclean) [reply ci/lre/Jenkinsfile:133]: To get the latest stable commit hash

File: ci/lre/Jenkinsfile (hunk 1 above)
Comment (mbenmclean) [reply ci/lre/Jenkinsfile:194]: It wasn't it's still there further down. Think it's just a little confusing with how github is rendering the diff

Comment (bobdonat) [reply ci/lre/Jenkinsfile:194]: I see this:

       stage('Upgrade to stable commit') {
            steps {
                script {
 with the "when" missing

Comment (bobdonat) [reply ci/lre/Jenkinsfile:194]: If you view the full file you will see it is missing

Comment (mbenmclean) [reply ci/lre/Jenkinsfile]: Fixed

File: ci/lre/Jenkinsfile (hunk 2 above)
Comment (bobdonat) [reply ci/lre/Jenkinsfile:133]: I think it is only needed when doing upgrade. The "when" could be added here.

Comment (mbenmclean) [reply ci/lre/Jenkinsfile:133]: fixed

**Review by davisgomes — APPROVED** [review]

**Pull request merged**